src/calculateDeepEqualDiffs.js

```diff
@@ -46,6 +46,14 @@ function accumulateDeepEqualDiffs(a, b, diffsAccumulator, pathString = ''){
       trackDiff(a, b, diffsAccumulator, pathString, diffTypes.different)
   }
 
+  if(isSet(a) && isSet(b)){
+    if(a.size !== b.size){return trackDiff(new Set(a), new Set(b), diffsAccumulator, pathString, diffTypes.different)}
+    for(const valA of a) 
+      {if(!b.has(valA)) 
```

Comment:
this is where you report they are different and not `deepEquals`

they are `deepEquals` if the for loop passes